### DISCUSSION TITLE
Data Privacy question.

### DISCUSSION THREAD

**Discussion by HeadClot** [upscayl/upscayl#782] [Q&A]
Hey - I am looking at using upscayl for up scaling blueprints for old buildings that have been digitized before the year 2000. Does upscayl capture any usage data? Does it send any data back of what I have been using with upscayl?

**Comment by NayamAmarshe** [comment]
No, we don't do that.
Even if we did, I don't think we'd be stealing the users' images. Telemetry is helpful sometimes, to make better design decisions but there's no data-stealing going on.

**Reply by HeadClot** [reply]
This answers my question Thank you.
